controllers/front/SitemapController.php

```diff
@@ -66,13 +66,8 @@ protected function getPagesLinks()
         $links = array();
```

Comment:
If we set a new value to $link a few lines later, I think we can remove this one.